plaidml/bridge/openvino/ops/reverse_sequence.cpp

```diff
@@ -0,0 +1,75 @@
+// Copyright (C) 2020 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "ngraph/opsets/opset.hpp"
+#include "ngraph/opsets/opset1.hpp"
+#include "plaidml/op/op.h"
+#include "plaidml_ops.hpp"
+#include "plaidml_util.hpp"
+
+using namespace plaidml;  // NOLINT[build/namespaces]
+using ngraph::opset1::ReverseSequence;
+
+namespace {
+
+template <typename T>
+std::vector<T> cast_constant_operand(size_t operand_idx, ngraph::Node* layer) {
+  auto* ngraph_const = ngraph::as_type<ngraph::op::Constant>(layer->get_input_node_ptr(operand_idx));
+  if (ngraph_const) {
+    return ngraph_const->cast_vector<T>();
+  } else {
+    THROW_IE_EXCEPTION << " input [1] is Unsupported inputType; ";
+  }
+}
+
+edsl::Tensor reverse_tensor(edsl::Tensor reverse_crop, int64_t seq_axis) {
+  std::vector<edsl::TensorDim> dims(reverse_crop.rank());
+  reverse_crop.bind_dims(dims);
+  std::vector<edsl::TensorIndex> I_idxs(reverse_crop.rank());
+  std::vector<edsl::TensorIndex> O_idxs;
```

Comment:
I left a comment here, I think we can leave it as it is. we are using std::vector in many other ops anyway.